src/main/java/jbash/commands/CmdMkdir.java

```diff
@@ -16,15 +15,15 @@ public String getHelp() {
     }
 
     @Override
-    public int execute(List<String> args) {
-        if (args.isEmpty()) {
-            System.out.println("mkdir: missing operand");
-            System.out.println("Try 'mkdir --help' for more information." );
+    public int execute(List<String> argv) {
+        if (argv.isEmpty()) {
+            cmdErrln("mkdir: missing operand");
+            cmdErrln("Try 'mkdir --help' for more information." );
         }
         FileSystemAPI FSAPI = FileSystemAPI.getInstance();
-        for (String arg : args) {
+        for (String arg : argv) {
             if (!FSAPI.createDirectory(arg)) {
```

Comment:
if we change createDirectory to return a Directory reference (or probably an optional) instead of a bool, we'll have to remember to change these also

Reply:
Yeah very true. We'll also need to update a bunch of tests too.